octodns_transip/__init__.py

```diff
@@ -116,12 +201,40 @@ def populate(self, zone, target=False, lenient=False):
                         lenient=lenient,
                     )
                     zone.add_record(record, lenient=lenient)
+
         self.log.info(
             'populate:   found %s records', len(zone.records) - before
         )
 
         return True
 
+    def _process_desired_zone(self, desired):
+
+        for record in desired.records:
+            if record._type == 'NS' and record.name == '':
+
+                # Check values for FQDN, IP's are not supported
+                values = record.values if record.values else [record.value]
```

Comment:
I believe `record.values` should be sufficent here since the `_type` is being checked above and `NS` records should always have the values property.

```suggestion
                values = record.values
```